backends/advanced/src/advanced_omi_backend/services/memory/config.py

```
@@ -122,6 +124,23 @@ def create_openmemory_config(
     }
 
 
+def create_mycelia_config(
+    api_url: str = "http://localhost:8080",
+    api_key: str = None,
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)